flask_resources/errors.py

```diff
@@ -8,56 +8,63 @@
 """Exceptions used in Flask Resources module."""
 
 import json
-from functools import wraps
+from functools import update_wrapper, wraps
 
 from flask import g
 from werkzeug.exceptions import HTTPException
 
 from .context import resource_requestctx
 
 
-def error_handler(f):
-    """Decorator that handles error serialization from here on out."""
+def create_error_handler(map_exception):
```

Comment:
I would name it something like ``create_errormap_handler`` or similar. The other name for me is too generic.